src/udfs/fastrcnn_object_detector.py

```diff
@@ -105,12 +105,25 @@ def _get_predictions(self, frames: Tensor) -> List[Outcome]:
             pred_boxes = list(pred_boxes[:pred_t + 1])
             pred_class = list(pred_class[:pred_t + 1])
             pred_score = list(pred_score[:pred_t + 1])
-            prediction_df_list.append(
-                Outcome(pd.DataFrame(
-                    {"label": pred_class, "pred_score": pred_score,
-                     "pred_boxes": pred_boxes}),
-                    'label'))
-        return prediction_df_list
+            outcome = outcome.append(
+                {
+                    "label": pred_class,
+                    "pred_score": pred_score,
+                    "pred_boxes": pred_boxes
+                },
+                ignore_index=True)
+        return outcome
+
+    def classify(self, frames: Tensor) -> pd.DataFrame:
+        gpu_batch_size = ConfigurationManager()\
+            .get_value('executor', 'gpu_batch_size')
 
-    def classify(self, frames: Tensor) -> List[Outcome]:
-        return self._get_predictions(frames)
+        if gpu_batch_size:
```

Comment:
Can we move this to the abstract class so that it happens automatically for every UDF?

Reply:
Done in fe148be